src/test/java/com/alexbro/onlinebank/core/service/account/DefaultAccountOperationServiceTest.java

```diff
@@ -42,4 +48,18 @@ public void shouldTransfer() {
         verify(accountDao).update(accountFrom);
         verify(accountDao).update(accountTo);
     }
+
+    @Test
+    public void shouldGetSumAfterExchange(){
+       BigDecimal result = testedInstance.getSumAfterExchange(BigDecimal.valueOf(SUM), BigDecimal.valueOf(CURRENT_RATE), BigDecimal.valueOf(EXCHANGE_RATE));
```

Comment:
Why don't you create a constant of BigDecimal type instead of double? There are 3 useless code duplications